cli/src/commands/git/push.rs

```diff
@@ -484,7 +492,7 @@ fn update_change_branches(
         if view.get_local_branch(&branch_name).is_absent() {
             writeln!(
                 ui.status(),
-                "Creating branch {branch_name} for revision {change_arg}",
+                "Creating branch {branch_name} for revision {short_change_id}",
```

Comment:
nit: this change is quite unrelated to the `all:` bit, so it should ideally be its own commit